bangazonapi/views/product.py

```diff
@@ -104,12 +105,16 @@ def create(self, request):
 
             new_product.image_path = data
 
-        new_product.save()
+        try:
+            new_product.save()
 
-        serializer = ProductSerializer(
+            serializer = ProductSerializer(
             new_product, context={'request': request})
+            return Response(serializer.data)
+
+        except ValidationError as ex:
 
-        return Response(serializer.data, status=status.HTTP_201_CREATED)
+         return Response(serializer.data, status=status.HTTP_400_BAD_REQUEST)
```

Comment:
looks like you have a space before this return. Fix that before you merge, please.